api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/context/CompletePaymentMessageContext.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.dhsc.htbhf.claimant.message.context;
+
+import lombok.Builder;
+import lombok.Value;
+import uk.gov.dhsc.htbhf.claimant.entity.Payment;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+
+@Value
+@Builder
+public class CompletePaymentMessageContext {
+    private PaymentCycle paymentCycle;
+    private Payment payment;
+    private String referenceId;
```

Comment:
I think this `referenceId` should have a more specific name as we have quite a lot of reference ids in the the system, how about `paymentProviderReferenceId` or just `paymentReferenceId`?

Reply:
`paymentReferenceId` gets my vote

Reply:
How about depositFundsReferenceId

Reply:
Changed to paymentReferenceId